src/main/java/bio/terra/app/logging/LoggerInterceptor.java

```diff
@@ -47,7 +68,8 @@ public void afterCompletion(
           url,
           method,
           paramString,
-          responseStatus);
+          responseStatus,
+          stackDriverPayload);
```

Comment:
If we're just logging as text, we should just add the format string here too to show the post body

Reply:
And many put it at the end preceded with a newline so it's a little easier to view in the stackdriver logs?

Reply:
I'm hesitant to put it into the format string. For the sake of readability, its much easier to include it in the stackdriver payload and let the stackdriver ui hide it. We can still search on it, but it won't clutter the logs.

Reply:
Fair enough.  Do you happen to have an example of the logs with this enabled?

Reply:
![image](https://user-images.githubusercontent.com/3210510/148074101-8ede7b7c-9dcb-49e5-bb35-92a2aaedd139.png)